clinical-clusters/2019/clinical-clusters.json

```diff
@@ -1012,37 +1157,39 @@
     ]
   },
   {
-    "measureId": "185",
-    "submissionMethod": "claims",
+    "measureId": "102",
+    "submissionMethod": "registry",
     "firstPerformanceYear": 2017,
     "lastPerformanceYear": null,
-    "clinicalClusters": [
+    "specialtySets": [
       {
-        "name": "endoscopyAndPolypSurveillance",
+        "name": "radiationOncology",
         "measureIds": [
-          "185",
-          "320"
+          "102",
+          "143",
+          "144"
         ]
       }
     ]
   },
   {
-    "measureId": "320",
-    "submissionMethod": "claims",
+    "measureId": "143",
+    "submissionMethod": "registry",
     "firstPerformanceYear": 2017,
     "lastPerformanceYear": null,
-    "clinicalClusters": [
+    "specialtySets": [
       {
-        "name": "endoscopyAndPolypSurveillance",
+        "name": "radiationOncology",
         "measureIds": [
-          "185",
-          "320"
+          "102",
+          "143",
+          "144"
```

Comment:
According to https://confluence.cms.gov/pages/viewpage.action?spaceKey=QPPPRACTICE&title=EMA+Logic+Year+3 it looks like `radiationOncology ` was supposed to be removed. Might be some context I don't know...

Reply:
the ema cluster builder reorders everything so git thinks it was updated, this one is actually  for:  Radiation Oncology* | Registry rather then Claims, if you scroll to the bottom of the speciality scenarios you can see that one for measures 102, 143, 144

Reply:
if you look for claims for 156 the whole measure is no longer in the list...